File: src/targets/node/fetch/fixtures/application-form-encoded.js
const { URLSearchParams } = require('url');
const fetch = require('node-fetch');

const encodedParams = new URLSearchParams();
encodedParams.set('foo', 'bar');
encodedParams.set('hello', 'world');

<<<<<<< HEAD
const url = 'https://httpbin.org/anything';
=======
const url = 'http://mockbin.com/har';
>>>>>>> upstream/master
const options = {
  method: 'POST',
  headers: {'content-type': 'application/x-www-form-urlencoded'},
  body: encodedParams
};

<<<<<<< HEAD
fetch(url, options)
  .then(res => res.json())
  .then(json => console.log(json))
  .catch(err => console.error('error:' + err));
=======
try {
  const response = await fetch(url, options);
  const data = await response.json();
  console.log(data);
} catch (error) {
  console.error(error);
}
>>>>>>> upstream/master

File: src/targets/ruby/faraday/fixtures/application-form-encoded.rb
require 'faraday'

data = {
  :foo => "bar",
  :hello => "world",
}

conn = Faraday.new(
  url: 'https://httpbin.org',
  headers: {'Content-Type' => 'application/x-www-form-urlencoded'}
)

response = conn.post('/anything') do |req|
  req.body = URI.encode_www_form(data)
end

puts response.status
puts response.body